### ISSUE TITLE
[FE] 10-10EZ - Investigate compatibility with Read Aloud text-to-speech extension

### ISSUE THREAD

**Issue by aagosto90** [department-of-veterans-affairs/va.gov-team#82009]
## Benefit to the Veteran
For Veterans who are not 100% blind but may have low vision, cognitive issues, and other conditions, there are text-to-speech extensions that can be downloaded for free to read a web page aloud, such as the Read Aloud Google Chrome extension. 

## Background
In usability testing for TERA in April, our one participant who said they were a screen reader user wasn't able to use their chosen screen reader software for the session. They were trying to use ["Read Aloud"](https://chromewebstore.google.com/detail/read-aloud-a-text-to-spee/hdhinadidafjejdhmfkjgnolgimiaplp) but the extension was only showing the intro page content no matter where they were in the form. This ticket is to investigate why that might be happening and determine if there's anything we can do on our end to enable that software to read the current page of the form.

Finding about this screen reader issue:
> Our one participant who uses a screen reader extension in Chrome was not able to use it for the session because the extension (called Read Aloud) was showing text from the intro page no matter where they were in the form. They ended up magnifying the page to complete the session.

Once it's determined if there's a fix for this, we can make another ticket for that work if it's going to take any significant amount of time. 

## Tasks
- [x] Do some discovery work with the Read Aloud extension to determine why it's fetching intro page content
     - Including reaching out to other teams to see if they ran into anything similar
- [x] Let the rest of the EZ team know what your findings are and whether we need to create any subsequent tickets for this work.

## Acceptance Criteria
- We've done all that we can to investigate the Read Aloud extension issues and have determined whether we can make it work for all pages of the 10-10EZ or not.

**Comment by longmd** [comment]
After spending some time with this today, here are my main findings:

1. This extension does not actively watch/listen for any navigation. It scans your current page and reads the text until it reaches the end of what it scanned or you tell it to stop.
2. If you navigate to another page while it is still reading the previous page, you must stop the reading and tell it to read your current page. Even in their product description in the Chrome store it says "Read aloud the _current_ webpage article with one click, using text to speech (TTS)"
3. This extension cannot read the text in things like buttons or web components--it looks strictly for text, making it different from actual screenreader software. This can make things awkward if you're reading along with the speech and see it skip various parts of the page.

The key takeaway is the behavior described from the session matches the behavior I experienced with the extension active. I tried this on various other sites outside of the VA (like news, sports, etc.) and found the behavior to be the same as what I found when going through our apps and other parts of VA.gov. I think this is nothing more than a limitation of the extension itself and its doing what they say its designed to do.

cc: @hdjustice
